frontend/src/metabase/lib/expressions/lexer.js

```diff
@@ -196,19 +196,27 @@ const getQuoteCategories = character => {
     : [];
 };
 
+const quotedStringRegex = (character, closed = true) => {
+  const open = character;
+  const close = closed ? character : "";
+  return new RegExp(
+    `${open}(?:[^\\\\${character}]|\\\\(?:[bfnrtv${character}\\\\/]|u[0-9a-fA-F]{4}))*${close}`,
```

Comment:
Nice fix! "catastrophic backtracking" I guess? What's the `bfnrtv` part?

Reply:
Those are other characters that can be escaped, e.x. `\n`. I copied it from Chevrotain's JSON lexer since we just use `JSON.stringify`/`parse` to escape/unescape string literals. I think they must have fixed the regex since I first copied it.